src/core/ext/filters/client_channel/lb_policy.cc

```diff
@@ -91,6 +91,29 @@ LoadBalancingPolicy::UpdateArgs& LoadBalancingPolicy::UpdateArgs::operator=(
   return *this;
 }
 
+//
+// LoadBalancingPolicy::PickResult
+//
+
+LoadBalancingPolicy::PickResult LoadBalancingPolicy::PickResult::Complete(
```

Comment:
Question: do we still want these API's, given that we can now unambiguously construct a type with the right fields?

Reply:
Good idea, done.